tempesta_db/core/main.c

```diff
@@ -48,6 +51,20 @@ tdb_entry_create(TDB *db, unsigned long key, void *data, size_t *len)
 EXPORT_SYMBOL(tdb_entry_create);
 
 /**
+ * Create TDB entry to store @len bytes.
+ */
+TdbRec *
+tdb_entry_alloc(TDB *db, unsigned long key, size_t *len)
+{
+	TdbRec *r = tdb_htrie_insert(db->hdr, key, NULL, len);
+	if (!r)
+		TDB_ERR("Cannot create cache entry for key=%#lx\n", key);
```

Comment:
Please replace "create" with "allocate", otherwise the error is undistinguishable from `tdb_entry_create()`'s one.